src/Core/src/Graphics/ShapeDrawable.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace Microsoft.Maui.Graphics
```

Comment:
Isn't this confusing between the Graphics nupkg namespace and MAUI core namespace?